packages/amplication-data-service-generator/src/plugin-wrapper.ts

```diff
@@ -91,7 +92,7 @@ const pluginWrapper: PluginWrapper = async (
         error
       )}`,
     });
-    return [];
+    return context.utils.abortGeneration ? process.exit(0) : [];
```

Comment:
I think we should abort more gracefully and add a message to the logger

instead of using a boolean, we can use a function context.abortGeneration(message)